### ISSUE TITLE
Original text display styles

### ISSUE THREAD

**Issue by iG8R** [immersive-translate/immersive-translate#1441]
### 您希望的更新和改进是什么 | Update or Improve

Could you please add display styles for the original text on the page, similar to the `Translation display style` option for those who use immersive-translate in language learning.

Also it would also be great to have the style that allow to hide/bring up (fold/unfold) both original and translated text.
For example:

Normal style
```
Original text              or           Translated text
Translated text                         Original text
```

Style with hiding the text
By clicking on `+/-` or other designated symbols the text will be unfolded/folded
```
+                          or           Translated text
Translated text                         +
```
"click"
```
- Original text            or           Translated text
Translated text                         - Original text
```

**Comment by keyboard3** [comment]
![image](https://github.com/immersive-translate/immersive-translate/assets/13177744/b1f89ee4-a433-48d8-b171-24fb56ddc704)
You can try the above two functions, hide the original text can be switched by button or shortcut keys, individual paragraphs can also be switched by hovering the mouse

**Comment by iG8R** [comment]
Yes, I know about these options.
`Show translation only` is a very useful option when the user needs to see only translated text without additionally having to view the original text, but in a case like I aforementioned - it is not suitable.
As for the `hover` function, it is way inconvenient and I was forced to turn it off, because it interferes with many keyboard shortcuts. What I mean - very often, when the cursor leaves hovering over a paragraph and in this time I use a keyboard shortcut, e.g. to switch to the next tab (Ctrl+Tab), it triggers this function. And so it turns out that I very often use a variety of keyboard shortcuts, which include all possible key options, because no matter what settings I make for this function, it always works at the wrong moment. If I use `Custom shortcut key`, then I ended up with something like `Ctrl+Alt+Shift+Win+Z`, because almost all other convenient combinations are occupied, which is not very convenient to use.
But, if it would be a symbol that the user could click on to show the text - this way it would be much more convenient.
